feat(17328): Add manual reload to the event log

vanch3d opened the pull request:
See https://hivemq.kanbanize.com/ctrl_board/57/cards/17328/details/

This PR replaces the 5s polling of the events request by a user-triggered refetch mechanism. 

### Before
![screenshot-localhost_3000-2023 11 02-13_48_20](https://github.com/hivemq/hivemq-edge/assets/2743481/e3740aff-63ab-4a6b-b4a5-d55923bbd1df)


### After
![screenshot-localhost_3000-2023 11 02-13_48_05](https://github.com/hivemq/hivemq-edge/assets/2743481/0c1710c1-98bc-4f2a-859f-32face7449ba)

sfrehse reviewed (commented):
Do you think it makes sense to add a spinner to the left side of the button while loading?

vanch3d commented:
> Do you think it makes sense to add a spinner to the left side of the button while loading?
Added with the latest commit to the PR. And fixed the loading text too.

![Screenshot 2023-11-03 at 12 28 25](https://github.com/hivemq/hivemq-edge/assets/2743481/f754af2e-4627-4070-bebc-654f98d874da)

![Screenshot 2023-11-03 at 12 28 35](https://github.com/hivemq/hivemq-edge/assets/2743481/0e6ffd3a-e77e-4ecf-bbff-27a7e4b474ed)
